image_proc/src/libimage_proc/processor.cpp

```diff
@@ -1,130 +0,0 @@
-/*********************************************************************
```

Comment:
Can anyone comment as to why this file was removed?

In ROS 1, [the implementation is used by stereo_image_proc](https://github.com/ros-perception/image_pipeline/blob/00e7c02aeb3fe4283e6327e28eb9e24f0120fd98/stereo_image_proc/include/stereo_image_proc/processor.h#L168). I've opened a PR to add back the implementation https://github.com/ros-perception/image_pipeline/pull/484